src/components/markdown/index.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+import rehypeReact from "rehype-react";
+import styled from "styled-components";
+
+const PrimaryTitle = styled.h1`
+    font-weight: 400;
+    font-size: 2em;
```

Comment:
Should we use `rem`? Are we using `em` intentionally?

Reply:
I prefer `em` over `rem` for this kind of projects where you need granular control on elements size. 
Anyway, I noticed we used `pt`, `px`, `rem` and `em` on this project. 

We should adopt a strategy and stick to it, but I think this should be discussed in a separate PR.